tests/test_server.py

```diff
@@ -560,8 +539,61 @@ def test_pandadf_dump_flat(self):
         self.assertTrue("post_mean" in out_df.columns)
         self.assertTrue("post_var" in out_df.columns)
 
+    """
+    Tests that test the server connections, as well as message routing
+    """
+
+    def test_receive(self):
+        """test_receive - verifies the receive is working when server receives unexpected messages"""
+
+        message1 = b"\x16\x03\x01\x00\xaf\x01\x00\x00\xab\x03\x03\xa9\x80\xcc"  # invalid message
```

Comment:
Are these messages just gibberish?

Reply:
Yep, this was an old test I had to modify slightly.